README.md

```diff
@@ -1,2 +1,19 @@
-# lab-guidelines
-How-to guides for SSL projects and documents
+# Secure Systems Lab Guidelines
+
+Welcome to the Secure Systems Lab (SSL) Guidelines repo! Below you find
+relevant guides and how-tos for working and collaborating in the lab.
+
+* On-boarding -- You are new to SSL and its operations?
+  The [on-boarding guide](on-boarding.md) helps you get up to speed.
+* Starting new projects -- Follow our [new projects guide](newproject.md)
+  to start your project on the right foot.
+* Reporting progress -- Document your achievings and share your insights
+  by following the [progress report guide](progressreport.md).
+* Writing LaTeX documents -- Clear communication is key at SSL, and
+  our [LaTeX guide](latexdocuments.md) helps you master science's Number 1
```

Comment:
Why is `Number 1` capitalized?

Reply:
I was thinking of something among the lines of "The Number 1 Used Cars Dealer" which looked typographically idiomatic to me. I'm fine either way.